src/App.js

```diff
@@ -129,12 +138,12 @@ export default class App extends React.Component {
         fade,
       }));
 
-    return { nodes, maxDepth: root.height };
+    return { nodes };
```

Comment:
Probably doesn't need to return an object any more? Though it might make sense to cap `maxDepth` as `rootHeight - minDepth`.